edge-modules/TwinTester/docker/linux/arm64v8/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG base_tag=1.0.4-linux-arm64v8
```

Comment:
update base tag?

Reply:
Again, nope we stop supporting ARM64.

Reply:
We need to change as the old base image is deleted.